### DISCUSSION TITLE
How to handle redirects defined in WP?

### DISCUSSION THREAD

**Discussion by traed** [wpengine/faustjs#1878] [Q&A]
Using the plugin Redirection we can easily define redirects from a source URL to a target URL in WP admin. But this doesn't seem to have any effect since it's the frontend that handles routing of the request. What is the suggested way of handling redirects from the frontend side?

**Comment by stupart** [comment]
Did you find the solution to this problem @traed?

**Comment by traed** [comment]
We ended up using `getStaticProps` (or `getServerSideProps`) in `[...wordPressNode].tsx` to catch any 404, then get the redirect items from GQL, check if there's matching one and if yes, redirect. I would've loved to have this built in to faust though.

**Reply by stupart** [reply]
Thanks @traed !
